src/main/resources/application-worker.yaml

```
@@ -24,3 +24,9 @@ rhsm-subscriptions:
     keystore: file:${SUBSCRIPTION_KEYSTORE:}
     keystorePassword: changeit
     maxConnections: ${SUBSCRIPTION_MAX_CONNECTIONS:100}
+  product:
+    useStub: ${STUB_PRODUCT:true}
+    url: http://${PRODUCT_HOST:localhost}:${PRODUCT_PORT:8080}/svcrest/product/v3
```

Let's just go ahead and default to something more useful for us:
```suggestion
    url: https://${PRODUCT_HOST:product.qa.api.redhat.com}:${PRODUCT_PORT:443}/svcrest/product/v3
```
Notably, this is *https*, and the QA endpoint with TLS port.

---

I thought the following would be even more useful, maybe?

```java
url: ${PRODUCT_URL:https://product.qa.api.redhat.com/svcrest/product/v3}
```

(this was also done for the subscription config)
If you disagree and there is more benefit to providing only the host and port, rather than the entire base URL, let me know and I'll use the provided suggestion verbatim.

---

:+1: